x-pack/plugins/security_solution/public/overview/components/entity_analytics/header/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useMemo } from 'react';
+import { EuiFlexGroup, EuiFlexItem, EuiPanel, EuiTitle } from '@elastic/eui';
+import styled from 'styled-components';
+import { useDispatch } from 'react-redux';
+import { useHostRiskScoreKpi, useUserRiskScoreKpi } from '../../../../risk_score/containers';
+import { LinkAnchor, useGetSecuritySolutionLinkProps } from '../../../../common/components/links';
+import { Direction, RiskScoreFields, RiskSeverity } from '../../../../../common/search_strategy';
+import * as i18n from './translations';
+import { getTabsOnHostsUrl } from '../../../../common/components/link_to/redirect_to_hosts';
+import { SecurityPageName } from '../../../../app/types';
+import { HostsTableType, HostsType } from '../../../../hosts/store/model';
+import { hostsActions } from '../../../../hosts/store';
+import { usersActions } from '../../../../users/store';
+import { getTabsOnUsersUrl } from '../../../../common/components/link_to/redirect_to_users';
+import { UsersTableType } from '../../../../users/store/model';
+import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
+
+const StyledEuiTitle = styled(EuiTitle)`
+  color: ${({ theme: { eui } }) => eui.euiColorVis9};
+`;
+
+export const EntityAnalyticsHeader = () => {
+  const { severityCount: hostsSeverityCount } = useHostRiskScoreKpi({});
+  const { severityCount: usersSeverityCount } = useUserRiskScoreKpi({});
+  const dispatch = useDispatch();
+  const getSecuritySolutionLinkProps = useGetSecuritySolutionLinkProps();
+  const riskyUsersFeatureEnabled = useIsExperimentalFeatureEnabled('riskyUsersEnabled');
+  const riskyHostsFeatureEnabled = useIsExperimentalFeatureEnabled('riskyHostsEnabled');
+
+  const [goToHostRiskTabFilterdByCritical, hostRiskTabUrl] = useMemo(() => {
+    const { onClick, href } = getSecuritySolutionLinkProps({
+      deepLinkId: SecurityPageName.hosts,
+      path: getTabsOnHostsUrl(HostsTableType.risk),
+      onClick: () => {
+        dispatch(
+          hostsActions.updateHostRiskScoreSeverityFilter({
+            severitySelection: [RiskSeverity.critical],
+            hostsType: HostsType.page,
+          })
+        );
+
+        dispatch(
+          hostsActions.updateHostRiskScoreSort({
+            sort: { field: RiskScoreFields.riskScore, direction: Direction.desc },
+            hostsType: HostsType.page,
+          })
+        );
+      },
+    });
+    return [onClick, href];
+  }, [dispatch, getSecuritySolutionLinkProps]);
+
+  const [goToUserRiskTabFilterdByCritical, userRiskTabUrl] = useMemo(() => {
```

Comment:
This url is getting generated incorrectly:
<img width="1098" alt="image" src="https://user-images.githubusercontent.com/28942857/184704382-8eb8a382-e02a-4ad5-91f7-a0e6a383c819.png">

It's pointing to a user name: `users/name/userRisk` instead of `users/userRisk`

Reply:
Thanks for cathing it!